org.eclipse.scanning.device/src/org/eclipse/scanning/device/composite/CompositeNexusScannable.java

```diff
@@ -39,11 +42,15 @@ public CompositeNexusScannable() {
 
 	@Override
 	public T getPosition() throws Exception {
+		/* Throwing an exception here causes the Scanning GUI to show no scannables.
 		throw new UnsupportedOperationException("A CompositeNexusScannable should only be used as a per-scan monitor");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)